tests/functional/main_test.go

```diff
@@ -58,7 +58,7 @@ func setup() {
 	}
 
 	hname := os.Getenv("HOSTNAME")
-	if env == "" {
+	if env == "" || env == "dev" {
```

Comment:
@subintp Is this change intentional?

Reply:
yes. Ref: https://github.com/razorpay/metro/commit/6f904b6c82bbe57a9854d50f4890dc0a28597f15#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R88